docker-compose.yml

```diff
@@ -31,21 +37,3 @@ services:
     restart: unless-stopped
     volumes:
```

Comment:
need health check

Reply:
```
app-1  | ** (ArgumentError) could not call Module.put_attribute/3 because the module Lynx.Repo.Migrations.CreateConfigs is already compiled
app-1  |     (elixir 1.16.1) lib/module.ex:2360: Module.assert_not_readonly!/2
app-1  |     (elixir 1.16.1) lib/module.ex:2041: Module.__put_attribute__/5
app-1  |     /app/lib/lynx-0.11.9/priv/repo/migrations/20230304233120_create_users.exs:6: (module)
app-1 exited with code 1
Aborting on container exit...
```

Reply:
Error also occures on main

Reply:
I think something like this shall do the health check for the app

```yaml
    ports:
      - '4000:4000'
    healthcheck:
      test: ["CMD", "curl", "--fail", "http://localhost:4000/_health"]
      interval: 30s
      retries: 3
```